src/ui/components/CertificatesManager/CertificatesManager.tsx

```
@@ -1,66 +1,105 @@
+
+type CertificatesManagerWrapperProps = {
+  children: ReactNode;
+};
+
+const CertificatesManagerWrapper = ({
+  children,
+}: CertificatesManagerWrapperProps) => (
+  <Box flexGrow={1} flexShrink={1}>
+    {children}
+  </Box>
+);
```

ESLint

[error] Declare only one React component per file

(react/no-multi-comp)